templates/web.socketed.template.yml

```diff
@@ -12,14 +12,14 @@ run:
         #!/bin/bash
         rm -rf /shared/nginx.http*.sock
   - replace:
-     filename: "/etc/nginx/conf.d/discourse.conf"
-     from: /listen 80;/
+     filename: "/etc/nginx/conf.d/outlets/server/http.conf"
+     from: /listen 80;(\nlisten \[::\]:80;)?/
```

Comment:
Do we still need to use a `replace` here? We could just overwrite the file.

Reply:
I do not have to use `replace`.

The next operation is also a `replace` to replace listen on port 443 with a listen on a socket. Unfortunately, that one is more complex because `https.conf` has a lot more content. I could probably be creative and replace that `replace` too, but I did not want to complicate further a template that is rarely used AFAIK.

I decided to continue to use `replace` here too to stay consistent.